configure_tls_protocol_version_and_ciphers.md

```diff
@@ -0,0 +1,18 @@
+# Configure TLS Protocol Version and Ciphers
+
+## SDK
+### How to configure TLS version
+Use `jdk.tls.client.protocols` system property to enforce TLS version. This property was introduced to JDK7 `7u95` and to JDK6 `6u121`.
```

Comment:
Can we say explicitly to use TLS 1.2 somewhere? like it is said in other SDK's?

So, to be sure your .NET code is using TLS1.2, you should configure your operating system to use TLS1.2 and disable TLS1.0 and TLS 1.1.